apps/native-component-list/screens/ImagePickerScreen.js

```diff
@@ -37,7 +33,8 @@ export default class ImagePickerScreen extends React.Component {
       }
     };
 
-    const showPicker = async () => {
+    const showPicker = async mediaTypes => {
+      await Permissions.askAsync(Permissions.CAMERA_ROLL);
```

Comment:
Do we need this on all platforms? What if the permission is not granted -- do we need to handle that on web?